example/lib/peripheral_details/components/peripheral_auto_test_view.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:flutter/material.dart';
+import 'package:blemulator_example/device_details/view/button_view.dart';
+
+class PeripheralAutoTestView extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Text('Auto test view');
```

Comment:
IMHO wrapping a blank text, without any formatting, etc. is not necessary

Reply:
Refactored this to a single Text in the parent view.